awx_collection/plugins/module_utils/controller_api.py

```
@@ -512,121 +584,168 @@ def make_request(self, method, endpoint, *args, **kwargs):
+            except (HTTPError) as he:
+                # ---- Retryable HTTP errors ----
+                if self.is_retryable(he.code, method, url.path):
+                    last_response = he.code
+                    # 403 block to check for database errror
+                    if he.code == 403:
+                        try:
+                            body = he.read()
+                            raw = body.decode('utf-8') if isinstance(body, bytes) else str(body)
+                            try:
+                                parsed = loads(body)
+                            except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)